### DISCUSSION TITLE
Where can I locate my favorite repositories from the GitHub app on the website?

### DISCUSSION THREAD

**Discussion by mdjdev** [community/community#121917] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,

I'm using the official GitHub Android app, which allows me to mark repositories as "favorites." I find this feature quite useful. However, I haven't been able to locate this functionality on the website. Is it possible that it doesn't exist on the website, or am I overlooking something?

**Comment by Vishwa-ud** [comment]


**Reply by mdjdev** [reply]
The favorite repositories are not listed as starred repositories. I believe these are separate functionalities.

**Comment by ebndev** [comment, marked as answer]
Hi @mdjdev, thanks for posting in the Community!

Favoriting a mobile only feature to add the repos to the main Homepage on the app. You might want to check out [Lists](https://github.blog/changelog/2022-06-21-lists-are-now-available-on-github-mobile-for-android/), available on Mobile and Web or organize starred repositories.

**Reply by alexcmb** [reply]
For me, a "star" is a sign of trust in a project. I’ve tested it, I like it, and I star it.
If I want to access a project easily from both my mobile and my computer, I just need it as a favorite but I don’t necessarily want to star it.
So the “favorite” feature in the GitHub app feels useless to me, and the list as well, because it doesn’t match the actual use case.

**Comment by eldarski** [comment]
### Favorites in GitHub iOS App

<ins>The GitHub iOS app has a separate "Favorites" functionality that is distinct from starring repositories:</ins>
**• Favorites are a mobile-only feature specifically for the GitHub app.**
• They allow you to add repositories to the main Homepage of the app for quick access.
• This feature is not synchronized with the web version of GitHub.
• Favorites are limited to 25 repositories.

So, basically, **Favorites** is UI feature for quick access within the mobile app only.

**Reply by j4hangir** [reply]
> ### Favorites in GitHub iOS App
> The GitHub iOS app has a separate "Favorites" functionality that is distinct from starring repositories: **• Favorites are a mobile-only feature specifically for the GitHub app.** • They allow you to add repositories to the main Homepage of the app for quick access. • This feature is not synchronized with the web version of GitHub. • Favorites are limited to 25 repositories.
> 
> So, basically, **Favorites** is UI feature for quick access within the mobile app only.

That's almost entirely useless then, what's the point of favoriting a repo when can't find it on desktop?

**Reply by jamessimpson007** [reply]
yeah seconded, how ridiculous

**Reply by phinnaeus** [reply]
Was using the iOS app to "forward" a bunch of repos to myself when I got back to my desk, so I didn't have to go through the many-click process of sharing their URLs into a note or something. Unbelievable to find out there's no way to access my favourites from desktop. Shambolic

**Reply by nickwshaw** [reply]
Really not useful

**Comment by aliber4079** [comment]
where do I submit an enhancement to have mobile stars syncd with web favorites

edit: oh this is what I needed

```https://github.com/<profilename>?tab=stars```
